### DISCUSSION TITLE
Cannot install AMS 2.5.3 due a non-existing package.

### DISCUSSION THREAD

**Discussion by clementgineste** [ant-media/Ant-Media-Server#5052] [Q&A]
Current [installer](https://gist.github.com/clementgoclock/f14e8e244b1734d098aa0dede24f41ca) stop installing AMS 2.5.3 (Entreprise) on a fresh and updated 20.04 **ARM** Ubuntu due a non-existing package : `libcrystalhd-dev`

line:248 : `$SUDO apt-get install libcrystalhd-dev -y`

Well there is no such package on Ubuntu 20.04. 

Universe repo actived.

**Comment by yashtandon113** [comment]
@clementgoclock 
Are you using an ARM 64 processor? If so, this package is not available for ARM. You can proceed by commenting it in a script as shown below.

`#$SUDO apt-get install libcrystalhd-dev -y`

I hope it helps.

**Comment by clementgineste** [comment]
Indeed, ARM. Forgot to add that characteristic. 
Thanks for your answer but in the mean time i used an old installer. 
I suggest that you can add a case switch for handling ARM CPU.

Thanks ;)
